### PR TITLE
#258 : Add redirect_uri and locale to request url to IMS

### PR DESCRIPTION
**Pull request by konarshankar07**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#258: Add redirect_uri and locale to request url to IMS

### CODE REVIEW

File: AdobeStockClient/Model/Config.php [hunk 1]
```diff
@@ -103,7 +114,11 @@ public function getAuthUrl(): string
         return str_replace(
             '#{client_id}',
             $this->getApiKey(),
-            $this->scopeConfig->getValue(self::XML_PATH_AUTH_URL_PATTERN)
+            str_replace(
```
Comment (sivaschenko) [inline AdobeStockClient/Model/Config.php]: Please apply str_replace only once using array arguments

**Review by sivaschenko — CHANGES_REQUESTED** [review]
@konarshankar07 thanks for the pull request! As I can see only redirerct_uri is added to the request, are you going to add locale as well? See my review comments.

File: AdobeStockClient/etc/config.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
                 <environment>PROD</environment>
                 <product_name>magento-adobe-stock-integration</product_name>
                 <token_url>https://ims-na1.adobelogin.com/ims/token</token_url>
-                <auth_url_pattern><![CDATA[https://ims-na1.adobelogin.com/ims/authorize?client_id=#{client_id}&scope=openid&response_type=code]]></auth_url_pattern>
+                <auth_url_pattern><![CDATA[https://ims-na1.adobelogin.com/ims/authorize?client_id=#{client_id}&redirect_uri=#{redirect_uri}&locale=en_US&scope=openid&response_type=code]]></auth_url_pattern>
```
Comment (konarshankar07) [inline AdobeStockClient/etc/config.xml]: I have added the locale here. Is it correct?

Comment (Den4ik) [reply AdobeStockClient/etc/config.xml]: @konarshankar07 Based on description here https://github.com/magento/adobe-stock-integration/issues/41 locale should takes from Magento configuration

Comment (konarshankar07) [reply AdobeStockClient/etc/config.xml]: @Den4ik 
Thanks for the clarification. I'll add the Magento configuration locale.

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**